packages/volto-slate/src/widgets/HtmlSlateWidget.jsx

```diff
@@ -127,7 +132,10 @@ const HtmlSlateWidget = (props) => {
             block={block}
             selected={selected}
             properties={properties}
+            extensions={slateWidgetExtensions}
```

Comment:
I fixed the issue with inconsistency,  htmlSlateWidget is used as `richtext` and we override this in our policy adddon. Thus, the same extension should be ported here as well.

I will write the cypress for this scenario.

Reply:
-cc @davisagli